t/Makefile.am

```diff
@@ -1,7 +1,6 @@
 # This list is included in both TESTS and dist_check_SCRIPTS.
 TESTSCRIPTS = \
 	t0000-sharness.t \
-	t1000-print-hierarchy.t \
```

Comment:
It looks like this file isn't deleted, even though it's removed from the testsuite?

Reply:
Oh yeah, I forgot to delete this file. Thanks for catching that